tslint.json

```diff
@@ -185,6 +185,7 @@
     "react-a11y-lang": true,
     "react-a11y-meta": true,
     "react-a11y-no-onchange": true,
+    "use-simple-attribute": true,
```

Comment:
Sorry to nitpick, but could you please keep these rules alphabetized? I find it convenient to be able to quickly scan through them.

Reply:
done.